docs/en/trust.rst

```diff
@@ -386,12 +386,12 @@ giving the references of the metadata protocol for each of these.
 Entity Statements
 -----------------
 
-Trust Anchors and Intermediates publish an Entity Statement related to a Subordinate.
+Trust Anchors and Intermediates publish an Entity Statement related to an immediate Subordinate.
```

Comment:
```suggestion
Trust Anchors and Intermediates publish an Entity Statement related to a direct Subordinate.
```

Reply:
usiamo immediate in oidc fed, uso lo stesso termine